js/Track/Model/File/BIGWIG.js

```diff
@@ -0,0 +1,19 @@
+Genoverse.Track.Model.File.BIGWIG = Genoverse.Track.Model.Graph.Bar.extend({
+
+  getData : function(chr, start, end){
+    var model = this;
+    var deferred = $.Deferred();
+
+    if(!this.url) this.data = new dallianceLib.BlobFetchable(this.track.dataFile);
+    else this.data = new dallianceLib.URLFetchable(this.url);
+
+    new BWReader(this.data, this.name, function(bw){
+      bw.getValues(chr, start, end, function(features){
```

Comment:
It is possible for bw to be null - need to check for this and immediately resolve the deferred if it is the case